readme.go

```diff
@@ -1,5 +1,8 @@
 package flatfs
 
+// TODO: now that datastore don't store CIDs but multihashes instead, this is really
```

Comment:
Found out later that it's addressed in https://github.com/ipfs/go-ds-flatfs/issues/103